### ISSUE TITLE
base_margin default in multiclass

### ISSUE THREAD

**Issue by BlindApe** [dmlc/xgboost#1380]
Seems now the default value for multiclass problem (softmax) is 1 / number of classes. This is specially painful when there are a high unevenly distribution of priors probabilities.

Could you set the default value to the priors (relative frequency of cases in each class)?
This would be more aligned with the default value for binary problem too (currently the mean of the response, not 0.5).
